app/assets/javascripts/controllers/middleware_topology/middleware_topology_controller.js

```diff
@@ -10,7 +10,7 @@ function MiddlewareTopologyCtrl($scope, $http, $interval, $location, topologySer
 
     $scope.refresh = function() {
         var id;
-        if ($location.absUrl().match("show/$") || $location.absUrl().match("show$")) {
+        if ($location.absUrl().match('show/$') || $location.absUrl().match('show$')) {
```

Comment:
I didn't want to end up in a state that new code will use the strings with single quotes and the old one the double quotes, so I did the auto replace. Sorry, that it makes the review more difficult, but for me the consistency is simply more.

btw.
https://github.com/ManageIQ/manageiq/issues/8781#issuecomment-221531748
https://github.com/hawkular/hawkular-client-ruby/pull/77#discussion_r67828729